src/foam/nanos/controller/ApplicationController.js

```diff
@@ -254,8 +254,9 @@ foam.CLASS({
         this.fetchTheme().then(() => {
           this
             .addClass(this.myClass())
-            .start('div', null, this.topNavigation_$)
-              .tag(this.TopNavigation)
+            .start()
+              .tag(this.topNavigation_$.map(
+                (s) => s ? foam.u2.ViewSpec.createView(s, {}, this, this.__subContext__) : 'NULL'))
```

Comment:
I think this should work. It keeps the existing behaviour of defaulting to the imported `TopNavigation`, and also removes the need to call ViewSpec.createView manually, since `tag` will do it for you.

```suggestion
                function(s) {
                  return this.E().tag(s || self.TopNavigation);
                }
```

Maybe we can get rid of the `map` altogether and just do something like

```
.tag(this.topNavigation_$)
```

but I'm not sure that'll work.

If `topNavigation_` isn't `class: 'foam.u2.ViewSpec'` then it should be updated to that as well. And it could have a factory that returns `this.TopNavigation`.

Reply:
So I've adjusted to use default. And made similar changes to the footerview.

But the suggestion doesn't work. Looks like because its an inner element, it doesn't size correctly in the ablii app, where the menu is suppose to vertically align.

![image](https://user-images.githubusercontent.com/10802216/67875953-879ef900-fb0d-11e9-9384-990a303d8e58.png)